android/gradlew.bat

```diff
@@ -0,0 +1,92 @@
+@rem
```

Comment:
Do we need files for Win?

Reply:
I wouldn't touch any of autogenerated scripts. Just to spend ages fixing it, if something breaks in the future.